File: 프로그래머스/0/120817. 배열의 평균값/배열의 평균값.java
class Solution {
    public double solution(int[] numbers) {
        double aver;
        int add=0;
        for(int i=0; i<numbers.length; i++){
            add+=numbers[i];
        }
        aver = (double)add/numbers.length;
        return aver;
    }
}

File: 프로그래머스/0/120817. 배열의 평균값/배열의 평균값.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// numbers_len은 배열 numbers의 길이입니다.
double solution(int numbers[], size_t numbers_len) {
    int add=0;
    
    for(int i=0;i<numbers_len;i++){
        add+=numbers[i];
    }
    
    return (double)add/numbers_len;
}